packages/app-config-writer/src/preview-config/preview-files.ts

```diff
@@ -2,12 +2,13 @@ import { basename, join } from 'path';
 import { getWebappPath } from '@sap-ux/project-access';
 import type { Editor } from 'mem-fs-editor';
 import type { ToolsLogger } from '@sap-ux/logger';
+import { TEST_CONFIG_DEFAULTS } from './ui5-yaml';
 
 const renameMessage = (filename: string): string =>
     `Renamed '${filename}' to '${filename.slice(
         0,
         -5
-    )}_old.html'. This file is no longer needed for the preview functionality. If you have not modified this file, you can delete it. If you have modified this file, move the modified content to a custom init script for the preview middleware. For more information, see https://www.npmjs.com/package/preview-middleware#migration.`;
+    )}_old.html'. This file is no longer needed for the preview functionality. If you have not modified this file, you can delete it. If you have modified this file, move the modified content to a custom init script for the preview middleware. For more information, see https://github.com/SAP/open-ux-tools/tree/main/packages/preview-middleware#migration.`;
```

Comment:
Just out of curiosity, why the change from npmjs.com to GitHub?

Reply:
The initial link was wrong and need to be adjusted. But then I noticed the deep link didn't work in npmjs readme whereas it worked in github. I always ended up at the very top of the readme in npmjs instead of the migration section. But as I just re-tested it now it seems to work again 🤷🏻
So if you prefer npmjs I can adjust the link as both seem to work now.